package.json

```diff
@@ -35,7 +35,7 @@
     "provinces": "^1.11.0",
     "regenerator-runtime": "^0.13.7",
     "register-service-worker": "^1.7.1",
-    "sbc-common-components": "2.4.19",
+    "sbc-common-components": "2.4.38",
```

Comment:
This was updated in another PR (still waiting to be merged). I will need to rebase.